Makefile

```
@@ -157,5 +169,6 @@ coverage:
 
 check: lint security test
 
-.PHONY: install lint format security snyk sonar graphs reports animations posters brand lighthouse \
+.PHONY: install lint format security snyk sonar graphs figure-contrast figures reports \
+	animations posters brand lighthouse \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)